lib/components/user/places/place-editor.tsx

```diff
@@ -144,6 +149,26 @@ class PlaceEditor extends Component<Props> {
               <ControlLabel htmlFor="name">
                 <FormattedMessage id="components.PlaceEditor.namePrompt" />
               </ControlLabel>
+              <div
```

Comment:
Consider adding `aria-hidden` to this block, so that the exceeded character limit isn't announced twice, once here and once in the help block below.

Reply:
Good idea, thank you!